<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDetailSecDlg</class>
 <widget class="QDialog" name="EditDetailSecDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>387</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Sql global</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Zona</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboZona"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Tabla</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboTabla"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QGroupBox" name="iGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Sql Interno</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="comboZonaInterna"/>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QComboBox" name="comboTablaInterna"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Sea igual a:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Tabla</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>de la tabla global.</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="comboFieldFrom"/>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboFieldTo"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Zona</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>de la tabla interna</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Seleccionar donde:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="chkColorear">
        <property name="text">
         <string>Colorear fondo</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QPushButton" name="btnColor1">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="chkAlternative">
        <property name="text">
         <string>Color Alternativo</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QPushButton" name="btnColor2">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAcept">
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="cabChk">
     <property name="text">
      <string>Cabecera</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nombre</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="footChk">
     <property name="text">
      <string>Pie</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
